portfolio/src/main/webapp/index.html

```diff
@@ -893,52 +893,52 @@
                     <span class="bimh"><div class="bimptri">▼</div><div class="bimp"><div class="muse">mcr</div></div>
                     
                     <div class="tbvolume">
-                    <script language="JavaScript">
-                        /*
-                        Random Image Script- By JavaScript Kit (http://www.javascriptkit.com)
-                        Over 400+ free JavaScripts here!
-                        Keep this notice intact please
-                        */
-                        function random_imglink(){
-                        var tbvol=new Array()
-                         
-                        tbvol[1]="up"
-                        tbvol[2]="down"
-                        tbvol[3]="off"
-                         
-                        var ry=Math.floor(Math.random()*tbvol.length)
-                        if (ry==0)
-                        ry=1
-                        document.write('<i class="fa fa-volume-'+tbvol[ry]+' "></i>')
-                        }
+                        <script language="JavaScript">
```

Comment:
You should ideally avoid adding any new javascript in the html file. Move it to the js file

Reply:
I moved the script, but it broke because of the document.write('<i class="fa fa-volume-"...) part of the code. Any thoughts on how I can fix this in the js file?